app/scripts/modules/netflix/pipeline/stage/canary/CanaryScore.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+
+import './canaryScore.component.less';
+
+interface ICanaryScoreProps {
+  score: number | string;
+  health?: string;
+  result: string;
+  inverse: boolean;
+}
+
+interface ICanaryScoreState {
+  score: number | string;
+  healthLabel: string;
+}
+
+export class CanaryScore extends React.Component<ICanaryScoreProps, ICanaryScoreState> {
+  public static defaultProps: Partial<ICanaryScoreProps> = {
+    health: ''
```

Comment:
default values for props, so the user of component doesn't have to pass in health, and it still will have a default value instead of undefined.